VERSION

```diff
@@ -1 +1 @@
-1.10.0
+1.11.0
```

Comment:
Can this be changed to use the version the way ZFS Local PV uses, automatically from github, so we don't need to keep pushing this change every release.

Reply:
Sure we can fetch it from github , as we don't have any upgrade code here which reads version from the VERSION file.
I will raise a different PR for automate version change